index.js

```diff
@@ -22,4 +27,20 @@ DefeatureifyFilter.prototype.processString = function (str) {
 	return defeatureify(str, this.options);
 };
 
+DefeatureifyFilter.prototype.cacheKeyProcessString = function cacheKeyProcessString(string, relativePath) {
+	return this.optionsHash() + crypto.createHash('md5').update(string, 'utf8').digest('hex');
+};
+
+DefeatureifyFilter.prototype.optionsHash  = function() {
+	if (!this._optionsHash) {
+		this._optionsHash = crypto.createHash('md5').update(stringify(this.options), 'utf8').digest('hex');
+	}
+
+	return this._optionsHash;
+};
```

Comment:
I don't see the point of this method. Hashing the options object shouldn't have any noticeable impact on perf.